app/javascript/controllers/stimulus_modules.js

```diff
@@ -0,0 +1,23 @@
+// Cross-site forgery protection is disabled in Rails' test environment and there are no tokens included in any
+// pages. Here we just send an empty string; however, if this were to outside the test environment, the protection
+// would be active and it would prevent the request.
+export function csrfToken () {
+  const node = document.querySelector('meta[name=csrf-token]')
+  if (node === null) {
+    return ''
+  } else {
+    return node.content
+  }
+}
+
+// Move to the last creator added to the form
+export function jumpToCreator () {
+  const nodes = document.getElementsByClassName('card-body')
```

Comment:
This seems really brittle, because this is a bootstrap class used for styling. If we decided to restyle these, it would break this javascript. Best bet here would be to create a new class for creator fields, and if you want to go full BEM style use a "js" prefix so someone coming in to tweak the html would know to think twice before changing it, like  `js-creator-fields`